contracts/arbitrators/KlerosLiquidProxy.sol

```diff
@@ -71,15 +71,10 @@ contract KlerosLiquidProxy is Multicall, IKlerosLiquidProxy {
             amountInUsd: claimAmountInUsd,
             evidence: evidence
         });
-        claimHashToDisputeId[claimHash] = disputeId;
+        claimHashToDisputeIdPlusOne[claimHash] = disputeId + 1;
         emit CreatedDispute(claimHash, claimant, disputeId);
-        emit Dispute(
-            klerosArbitrator,
-            disputeId,
-            META_EVIDENCE_ID,
-            uint256(claimHash)
-        );
-        emit Evidence(klerosArbitrator, uint256(claimHash), claimant, evidence);
+        emit Dispute(klerosArbitrator, disputeId, META_EVIDENCE_ID, disputeId);
+        emit Evidence(klerosArbitrator, disputeId, claimant, evidence);
```

Comment:
See https://github.com/kleros/kleros-interaction/blob/master/contracts/standard/arbitration/IArbitrable.sol
My understanding is that we can use the dispute ID as the evidence group ID